Issue 10416 - input-group buttons not sized properly

gskaplan opened the pull request:
This fix resolve the issue where input-group-buttons aren't filling up the entire space of the parent container. Thanks to @rafibomb for helping with the code. This relates to issue #10416.

Please review as necessary. @IamManchanda.

gskaplan committed:
Issue 10416 - input-group buttons not sized properly

Adjusted height for flex mode to fill parent container. Added two tests
- one for flex/xygrid and the other for flow mode.

IamManchanda commented:
Thanks digging it in!

IamManchanda commented:
@gskaplan Please rename `input-group-flow.html` to `input-group-float.html`

gskaplan committed:
Renaming test from input-group-flow to input-group-float

gskaplan commented:
Renamed and new PR created.

IamManchanda commented on scss/forms/_input-group.scss:
```diff
@@ -107,6 +107,7 @@ $input-prefix-padding: 1rem !default;
 
     @if $global-flexbox {
       flex: 0 0 auto;
+      display: flex;
```
Display first and then `flex` shorthand property 
See https://github.com/zurb/foundation-sites/blob/develop/.sass-lint.yml#L151-L153

gskaplan committed:
Merge remote-tracking branch 'refs/remotes/zurb/develop' into hotfix/input-group-buttons-sizing

Vinnl committed:
Re-order CSS properties

As suggested by @IamManchanda: https://github.com/zurb/foundation-sites/pull/10456#discussion_r128928162

gskaplan committed:
Merge pull request #1 from Vinnl/patch-2

Re-order CSS properties

IamManchanda commented:
Thanks... 
cc @kball

kball commented:
Excellent work. Thanks @gskaplan! And congratulations on first merged PR!
